docs/talks.rst

```diff
@@ -44,3 +44,14 @@ This list will be updated with talks given on :code:`pyhf`:
    :list: bullet
    :all:
    :style: plain
+
+
+Posters
```

Comment:
@kratsg @lukasheinrich Do you think it is enough to just add the Posters section or should we also change the top level index name to reflect that this isn't just a page of talks?

Reply:
I think talks includes "posters" too.